function_scope_lab.js

```diff
@@ -0,0 +1,304 @@
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal,
+//return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+console.log("----Q1----")
+
+const largerOutput = (num1, num2) => {
+  if(num1 > num2){
+    return num1 + " is larger"
+  } else {
+    return num2
+  }
+  }
+  let result1 = largerOutput(12, 12);
+  console.log(result1);
+
+  // ## Question 2
+  //
+  // Write a function named getDogAge that takes one argument: a dog's age
+  //in human years. The function returns the dog's age in dog years.
+  //The conversion rate is 1 human year to 7 dog years.
+  //
+  // Example:
+  //
+  // Input: `4`
+  //
+  // Output: `28`
+
+  console.log("-----Q2-----")
+  function getDogAge(dogAge) {
+    return dogAge * 7
+  }
+  let result2 = getDogAge(4);
+  console.log(result2)
+
+  // ## Question 3
+  //
+  // Write a function `arraySum` that takes in an array and returns the sum
+  //of all the elements.
+  //
+  // Example:
+  //
+  // Input: `[13,2,4,-2]`
+  //
+  // Output: `17`
+
+console.log("----Q3----")
+let arr1 = [13, 2, 4, -2]
+const arraySum = (arr1) => {
+let sum = 0
+for (let i = 0; i < arr1.length; i++) {
+    sum += arr1[i]
+  }
+ return sum
+  }
+  let result3 = arraySum(arr1);
+  console.log(result3);
+
+// ## Question 4
+//
+// Write a function that takes in any two numbers and return their sum.
+//
+// Example:
+//
+// Input: `14, 5`
+//
+// Output: `19`
+
+console.log("----Q4----")
+
+const totalNum = (num1, num2) => {
+  return num1 + num2
+}
+let result4 = totalNum(14, 5);
+console.log(result4);
+
+// ## Question 5
+//
+// Write a function takes in any number grade and returns a corresponding
+//letter grade.
+//
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+//
+//
+// Example:
+//
+// Input: `88`
+//
+// Output: `"B"`
+
+console.log('----Q5----')
+
+function grade(num) {
+  if (num === 100) {
+    return "A+"
+  } else if (num >= 90) {
+    return "A"
+  } else if (num >= 80) {
+    return "B"
+  } else if (num >= 70) {
+    return "C"
+  } else if (num >= 65) {
+    return "D"
+  } else {
+    return "F"
+  }
+  }
+  let result5 = grade(78);
+  console.log(result5);
+
+
+  // ## Question 6
+  //
+  // Implement a function named `repeatLog` that takes a string and a
+  //number as parameters. The function should log `message` `count`
+  //number of times.
+  //
+  // Example:
+  //
+  // Input: `"+", 10`
+  //
+  // Output: `++++++++++`
+
+console.log('----Q6----')
+const repeatLog = (num) => {
+  return str.repeat(num)
+}
+  let str = "+"
+  let result6 = str.repeat(10);
+console.log(result6);
+
```

Comment:
Great, right idea! This function should take in two arguments. Remember we need our code to be reusable
and easy. 

So if your repeatLog function took in (str, num) it would still work ! Plus you wouldn't have to keep redefining the variables. All you would have to do is call the function with the arguments. Ex: repeatLog("-",12)